### DISCUSSION TITLE
建议ppgw加挂载一个用于读写ppgw.ini的🍵本地盘

### DISCUSSION THREAD

**Discussion by dotjh** [kkkgo/PaoPaoGateWay#99] [想法]
首先感谢提供了一个这么好用的软件。我的需求很简单，通过dns对gfw分流到fakip。用adguardhome的upstream_dns_file就能进行简单分流。这样的好处是就是ppgw的节点失效了也不影响我gfw列表之外的访问。。现在就是觉得ppgw.ini的下载还要专门开个应用。。如果能在直接上传到ppgw本机就好了。再次感谢

**Comment by kkkgo** [comment]
1、用docker定制ISO可以内嵌ppgw.ini
2、配合PaoPaoDNS使用的话，有自带的http server
3、用docker定制ppgwurl.ini，可以定义从别的地方下载ppgw.ini，比如你可以把文件传到https://gist.github.com

**Reply by dotjh** [reply]
1、用docker定制了一个ppgwurl.ini的Iso。
2、docker了一个http-file-server.
感谢！
